src/styleguide-components/ComponentsList.js

```diff
@@ -1,80 +1,58 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import cx from 'clsx';
 import Link from './SidebarLink';
-import Styled from 'rsg-components/Styled';
+import styled from 'styled-components';
 
-const styles = ({ color, fontFamily, fontSize, space, mq }) => ({
-  list: {
-    margin: 0,
-    paddingLeft: '8px',
-  },
-  item: {
-    color: color.base,
-    display: 'block',
-    margin: [[space[1], 0, space[1], 0]],
-    fontFamily: fontFamily.base,
-    fontSize: fontSize.base,
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-  },
-  isChild: {
-    [mq.small]: {
-      display: 'inline-block',
-      margin: [[0, space[1], 0, 0]],
-    },
-  },
-  heading: {
-    color: color.base,
-    marginTop: space[1],
-    fontFamily: fontFamily.base,
-  },
-  headingLink: {
-    color: 'gray',
-    fontSize: 11,
-    textTransform: 'uppercase',
-    borderBottom: '1px solid #454545',
-    cursor: 'pointer',
-  },
-  link: {
-    color: 'whitesmoke',
-    cursor: 'pointer',
-    '&:hover': {
-      color: '#00B9A7',
-      cursor: 'pointer',
-    },
-  },
-});
+const LinkSpan = styled.span`
+  color: whitesmoke;
+  cursor: pointer;
+  &:hover {
+    color: #00b9a7;
+    cursor: pointer;
+  }
+`;
+
+const HeadingLinkSpan = styled.span`
+  color: ghostwhite;
+  font-weight: bold;
+  font-size: 18px;
+  letter-spacing: -0.5px;
+  margin-top: 12px;
+`;
+
+const List = styled.ul`
```

Comment:
We use this in shopfront as well might be nice as it's own component?

Reply:
This is for the web styleguide only. It's specific for react-styleguidist. More info: https://react-styleguidist.js.org/docs/cookbook.html#how-to-change-styles-of-a-style-guide